### ISSUE TITLE
Add ability to share a page with saved links

### ISSUE THREAD

**Issue by dmbaranov** [cnwangjie/better-onetab#151]
Hey there. Original OneTab has an option to share all the saved links as a webpage. So that I can easily check my saved tabs on other devices (e.g. smartphone). 

I wanted to implement this feature here myself, but I don't have access to the server, hence can't implement backend functionality. I see it in a way that the user has an ID (could be a token), he can open a link like https://boss.cnwangjie.com/token/tabs and see his saved tabs. 

It shouldn't be that difficult to implement, so you can either grant me access to the backend repository and I'll create 2 pull requests, or do it yourself if you have time. Thanks in advance :)

**Comment by bcbeneke** [comment]
I think the intent is to be able to select multiple pages or all pages using the checkboxes, select the three dot icon, and then use the copy link function to share a page. It does not correctly function, however. It only copies a string of text of the selected websites.

**Comment by dmbaranov** [comment]
Yes, that's what I'm talking about. Wouldn't it be more convenient to display all those links on a separate page?

**Comment by bcbeneke** [comment]
Yes, and it would also be nice to have the link automatically copied to the clipboard so there's less shuffling between tabs as well.

**Comment by dmbaranov** [comment]
Makes sense. Would you implement that? :)

**Comment by bcbeneke** [comment]
Would that I were able to. I only know how to complain!

**Comment by cnwangjie** [comment]
I'm sorry that I have not checked the issues in a long time because of busy. I think this is a useful feature for many people. I'm willing to add this feature in the future. I hope the style of the shared links page will be same as the extension to keep the style uniform.

Besides, the repo of the backend server is here:

https://github.com/cnwangjie/better-onetab-sync-server

It is great if you are willing to make a PR.

Thank you all. :laughing:

**Comment by dmbaranov** [comment]
Awesome, thank you! I currently don't have much time but will take a look once I do and will try come up with something :)

**Comment by waytootangled** [comment]
I REALLY need this feature or else i will have to revert back to OneTab!!

**Comment by nicbentulan** [comment]
(3 years later) have you done this please?

**Comment by nicbentulan** [comment]
https://github.com/cnwangjie/better-onetab/issues/255
